### PR TITLE
Build calculator UI display

### PR DESCRIPTION
**Pull request by acolombo1**
- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components and a new Calculator.js file inside.
- In Calculator.js, create a React component that matches the given design:
calculator

- Use only class-based components.
- Import the Calculator and use it in the main component (App.js).
- Test it in the browser.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,14 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+// eslint-disable-next-line react/prefer-stateless-function
+export default class App extends React.Component {
```
Comment (Gambit142) [inline src/App.js]: - [ ] Kindly remove all `eslint-disable` comments in all your JavaScript files. Linters help us write clean and maintainable codes, therefore using `eslint-disable` comments defeats the purpose of using linters when creating an application. Kindly remove all of them. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @acolombo1 👋, 

Good job so far!

## Highlights
- Used class-based components ✔️ 
- Created Calculator component ✔️ 
- Created App component ✔️

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- Kindly delete these files that were created after using the create react app command
  - public/favicon.ico
  - public/log192.png
  - public/logo512.png
  - public/manifest.json
  - public/robots.txt
  - src/App.test.js
  - src/reportWebVitals.js
  - src/setupTests.js

- _Kindly check the comments under the review for other required changes. 👍_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/App.js (hunk 1 above)
Comment (acolombo1) [reply src/App.js]: It does harm to me. The project asks to write classes, and the linter prefers stateless functions to classes. So I invent a state, but the linter requires it to be used. Then I invent a use, and the linter requires one more line to destructure what is not structured at all.

**Review by Lordkaito — APPROVED** [review]
Hi @acolombo1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**